app/services/benefit_check_service.rb

```diff
@@ -1,6 +1,19 @@
 class BenefitCheckService
   BENEFIT_CHECKER_NAMESPACE = 'https://lsc.gov.uk/benefitchecker/service/1.0/API_1.0_Check'.freeze
-  ApiError = Class.new(StandardError)
+  USE_MOCK = ActiveModel::Type::Boolean.new.cast(ENV['USE_BC_DEV_MOCK'])
```

Comment:
Shouldn't this `ENV` be read in `config/application.rb` like the other BenefitChecker related settings?
https://github.com/ministryofjustice/laa-apply-for-legal-aid/blob/master/config/application.rb#L20

Reply:
I wanted to limit the scope of this environment variable. It should never be used in production, so shouldn't suffer from the issues that make going through `Rails.configuration.x` worthwhile, and therefore passing it though `Rails.configuration.x` seems like an unnecessary complication.

Reply:
good point well made